src/components/AddWeddingForm/AddWeddingForm.tsx

```diff
@@ -36,8 +38,8 @@ const AddWeddingForm: React.FC<Props> = ({
     }
     const response = await postAWedding(newWedding);
     addNewWedding(response);
-    // should be a POST request + adding card to UI
     clearInputs();
+    history.push('/')
```

Comment:
I think this helps with the user experience - being able to be routed back to the main page after they add a wedding